src/aat/java/uk/gov/hmcts/ccd/datastore/tests/util/RetryUtility.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.ccd.datastore.tests.util;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.with;
+
+public class RetryUtility {
```

Comment:
shall we call this TestUtils so we can add more utility methods in the future?

Reply:
done